src/frontend/next/src/components/Login/LoggedOut.tsx

```diff
@@ -0,0 +1,33 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { Button } from '@material-ui/core';
+
+import useSiteMetadata from '../../hooks/use-site-metadata';
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    float: 'right',
+    margin: '0 0.5rem 0 0.5rem',
+  },
+  link: {
+    textDecoration: 'none',
+    fontSize: '1.5rem',
+    color: theme.palette.primary.contrastText,
+    lineHeight: 1,
+  },
+}));
+
+const LoggedOut = () => {
+  const { telescopeUrl } = useSiteMetadata();
+  const loginUrl = `${telescopeUrl}/auth/login`;
+  const classes = useStyles();
+
+  return (
+    <Button className={classes.button}>
+      <a href={loginUrl} className={classes.link}>
```

Comment:
You're not supposed to mix buttons and anchors like this.  It's one or the other.  In this case, I think we just want an anchor, since we redirect.  You can style it like a button.